doc/posts/zitification/ssh/zitifying-ssh.md

```diff
@@ -0,0 +1,133 @@
+client. In this configuration the `ssh` process is effectively "dark"
+meaning the service is not only invisible to the Internet but also the local area network as well! The only way to
+`ssh` to a machine using a [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network) will
+be to have an identity authorized for
+that [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network). Cool right? Let's see how
+we did that, and how you can do the same thing using
+a [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network).
+
+<hr>
+
+## How It's Done
+
+There are a few steps necessary before being able to use `zssh`:
+
+* Establish a [Ziti Network in AWS](https://github.com/openziti/ziti/blob/release-next/quickstart/aws.md) (using AWS in
+  this example)
+* Create and enroll two Ziti Endpoints (one for our `ssh` server, one for the client)
+    * the `sshd` server will run `ziti-tunnel` for this demonstration. Conviniently it will run on the same machine that
+      I used to setup the [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network).
+    * the client in this case will run `zssh` from my local machine and I'll `zssh` to the other endpoint
+* Create the [Ziti Service](https://openziti.github.io/ziti/services/overview.html) we'll use and authorize the two
+  endpoints to use this service
+* Use the `zssh` binary from the client side and the `ziti-tunnel` binary from the serving side to connect
+* Harden `sshd` further by removing port 22 from any Internet-based firewall configuration (for example from within the
+  security-groups wizard in AWS) or by forcing `sshd` to only listen on `localhost/127.0.0.1`
+
+After preforming these steps you'll have an `sshd` server which is dark to the Internet or your LAN (depending on how
+you decided to harden `sshd`). Accessing the server via `ssh` must now occur using the Ziti Network. Since the service
+is no longer accessible directly through a network it is no longer susceptible to the types of attacks mentioned
+previously!
+
+<hr>
+
+## Zssh in Action
+
+Once the prerequisites are satisfied, we can see `zssh` in action. Simply download the binary for your platform:
+
+* [linux](https://github.com/openziti-incubator/zssh/releases/download/latest-tag/zssh-linux-amd64)
+* [windows](https://github.com/openziti-incubator/zssh/releases/download/latest-tag/zssh-windows-amd64.exe)
+* [MacOs](https://github.com/openziti-incubator/zssh/releases/download/latest-tag/zssh-macos-amd64)
+
+Once you have the executable download, make sure it is named `zssh` and for simplicity's sake we'll assume it's on the
+path. The usage for `zssh` is very similar to the usage of `ssh` so anyone familiar with `zssh` should be able to pick
+up `zssh` easily, Should you forget the usage, executing the binary with no arguments returns the expected usage as
```

Comment:
Period instead of comma?

`up zssh easily, Should you forget the usage`

Reply:
reworked the whole paragraph